utils/extensions/models/notification.ts

```diff
@@ -3,70 +3,54 @@
  * Ideally they will be integrated in the common module
  */
 
-import * as DocumentDb from "documentdb";
 import { Either } from "fp-ts/lib/Either";
+import { TaskEither, tryCatch } from "fp-ts/lib/TaskEither";
 import {
   NotificationModel as NotificationModelBase,
   RetrievedNotification
 } from "io-functions-commons/dist/src/models/notification";
+import {
+  CosmosErrors,
+  toCosmosErrorResponse
+} from "io-functions-commons/dist/src/utils/cosmosdb_model";
+import { Errors } from "io-ts";
 import { NonEmptyString } from "italia-ts-commons/lib/strings";
-import * as DocumentDbUtils from "../documentdb";
 
 /**
  * Extends NotificationModel with deleting operations
  */
 export class NotificationDeletableModel extends NotificationModelBase {
-  /**
-   * Creates a new Notification model
-   *
-   * @param dbClient the DocumentDB client
-   * @param collectionUrl the collection URL
-   */
-  constructor(
-    dbClient: DocumentDb.DocumentClient,
-    collectionUrl: DocumentDbUtils.IDocumentDbCollectionUri
-  ) {
-    super(dbClient, collectionUrl);
-  }
-
   /**
    * Returns the notifications for the provided message id
    *
    * @param messageId The message the notifications refer to
    */
   public findNotificationsForMessage(
```

Comment:
can we use findNotificationsForMessage from io-fn-commons?

Reply:
Nope, see comment [https://github.com/pagopa/io-functions-admin/pull/65#discussion_r462458540](https://github.com/pagopa/io-functions-admin/pull/65#discussion_r462458540)

Reply:
same as above

Reply:
ping :-)